tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -360,18 +360,79 @@ Check created table has footnotes
     user checks list item contains    testid:footnotes    3    ${FOOTNOTE_ALL_FILTER}
 
 Save data block as a featured table
-    user enters text into element    id:dataBlockDetailsForm-name    UI Test data block name
-    user enters text into element    id:dataBlockDetailsForm-heading    UI Test table title
-    user enters text into element    id:dataBlockDetailsForm-source    UI Test source
+    user enters text into element    id:dataBlockDetailsForm-name    UI Test data block name 1
```

Comment:
Similarly to the `create_data_block_with_chart`, could use `label` selectors across this suite